mods/ra/weapons/ballistics.yaml

```diff
@@ -193,55 +125,29 @@ TurretGun:
 			Wood: 75
 			Light: 60
 			Heavy: 25
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
-	Warhead@3Eff: CreateEffect
-		Explosions: artillery_explosion
-		ImpactSounds: kaboom15.aud
-		InvalidImpactTypes: Water
+			Concrete: 100
 	Warhead@4EffWater: CreateEffect
 		Explosions: large_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 
 2Inch:
+	Inherits: ^Cannon
 	ReloadDelay: 60
 	Range: 5c512
 	Report: cannon2.aud
 	Projectile: Bullet
 		Speed: 426
-		Image: 120MM
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 25
 		Versus:
 			None: 30
-			Wood: 75
-			Light: 75
-			Concrete: 50
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
-	Warhead@3Eff: CreateEffect
-		Explosions: small_explosion
-		ImpactSounds: kaboom12.aud
-		InvalidImpactTypes: Water
-	Warhead@4EffWater: CreateEffect
-		Explosions: small_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 
 Grenade:
+	Inherits: ^Artillery
```

Comment:
This introduces a 3 cell minimum range, which is quite bogus considering that the maximum range is only 4 cells.